src/components/Home/Alert/index.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from '@dvcorg/gatsby-theme-iterative/src/components/Link'
+import React from 'react'
+import { cn } from '../../../utils'
+
+const banner = {
+  title: 'Expert Insights on Developing Safe, Secure, and Trustworthy AI!',
+  subtitle: 'Webinar | Wednesday, February 14th | 11 am ET',
+  link: 'https://www.linkedin.com/events/expertinsightsondevelopingsafe-7158518681906413569/about'
+}
+
+const Alert = () => (
+  <>
+    <div
+      className={cn(
+        'py-2',
+        'bg-orange text-white',
+        `w-full transition-[height] ease-in-out delay-75 duration-300 text-right px-2`
```

Comment:
There are added transition styles here, but I'm not seeing any animation. Is there supposed to be or do these styles need to be removed?

Reply:
It flashes for a moment from large to smaller on my screen.  Is that what this does?

Reply:
Sorry, that's from the previous alert component that Jeny mentioned that shows up late. I must have copied, but it was never used here.

> It flashes for a moment from large to smaller on my screen. Is that what this does?

That's because of font as website loads with default font and then applies the custom fonts later.